### DISCUSSION TITLE
Wavelog on Synology NAS

### DISCUSSION THREAD

**Discussion by HB9RYZ** [wavelog/wavelog#972] [Q&A] [answered]
Hello All
I have no idea how to install wavelog on my Synology NAS.
Is there a step by step installtion guide?
The latest Docker is installed on my Synlogy NAS.

I'm not a Linux expert and someone who can install Wavelog on my Docker.

73 de HB9RYZ
Wolfgang

**Comment by HB9HIL** [comment, marked as answer]
Hello Wolfgang

There is a written step by step install from one of our users :) 

https://mariushosting.com/how-to-install-wavelog-on-your-synology-nas/

Vy 73 de HB9HIL

**Reply by hellofinch** [reply]
BTW, **you should check your linux kernel first.** Follow [this](https://mariushosting.com/synology-how-to-check-linux-kernel-version/) guide.
If you found your linux kernel older than 4. You will fail to install.
![image](https://github.com/user-attachments/assets/74f7ac47-7698-40e4-a6fc-869f3fc8e70c)
Have a nice day, 73 de BI1UVZ.

**Reply by HB9RYZ** [reply]
> Hello Wolfgang
> 
> There is a written step by step install from one of our users :)
> 
> https://mariushosting.com/how-to-install-wavelog-on-your-synology-nas/
> 
> Vy 73 de HB9HIL

Hello Fabian - thank you very much. I will try this step-by-step installation quide on my Synology NAS. I will let know, if I had success to install it. 73 de HB9RYZ, Wolfgang

**Reply by HB9RYZ** [reply]
Hello - how to install portainer and how to config portainer is not writtn in the above step-by-step quide.
I installed portainer but I don't know how to configure portainer and how to access the portainer GUI.
There a few installation steps missing !

73' Wolfgang

**Comment by HB9RYZ** [comment]
> Hello All I have no idea how to install wavelog on my Synology NAS. Is there a step by step installtion guide? The latest Docker is installed on my Synlogy NAS.
> 
> I'm not a Linux expert and someone who can install Wavelog on my Docker.
> 
> 73 de HB9RYZ Wolfgang

**Comment by HB9RYZ** [comment]
![image](https://github.com/user-attachments/assets/ef77156f-c6ae-4f13-9282-cb8749fabadf)

wavelog will start and will stop after a few seconds !!!
Where is the problem?
Wieso ist dies alles immer so kompliziert ?

**Reply by HB9HIL** [reply]
This has primarly nothing to do with Wavelog. Maybe it's the kind of setup you try to install, some setting in Portainer or any other possible reason. 

Wavelog itself works like a basic PHP website and the only requirement is PHP (with the additional modules), a webserver and a database. There are many ways to deploy that and many more to fail. 

In fact running a webserver (with Wavelog or any other system) required some skills in Linux and connected systems which we, from Wavelog can not provide. There is no "setup.exe" which handles a complete deployment for you. With our docker-compose.yml we are pretty close to that but still you need some understandment how to run a webserver.

We are not responsible for any third-party install guides like the one for Synologie. So we only can offer limited support here.

As I already told you I can offer direct support as we both are HB9 and live in the same language area of switzerland :D 

I can fully understand that this was may a frustrating experience but don't give up.

**Reply by HB9RYZ** [reply]
Hello Fabian, you've described it well. It is clear that you cannot take responsibility for the peripheral systems or the operating system. And that is precisely the problem with Linux - there are far too many derivatives and too many different configurations. It's just a pity that good projects like “Wavelog” unfortunately fail because of the Linux environment when Hams who have no Linux knowledge try to get your solution up and running, as there are no good installation instructions. You have to ask Dr. Goggle for hours or spend hours in the forums. So I think a lot of Hams who like Wavelog would be willing to pay a fair amount for a Wavelog SaaS service. Because as you noted, I'm frustrated that after x hours of trying in vain to get the Linux environment to work for Wavelog, it still doesn't work. And Synolgy is not a solution that is unknown and used by many Hams. And once you have finally managed to get the Linux environment and Wavelog running, the question then arises as to how to update this thing and how to install further features - because then the whole disaster starts all over again. As long as I can't get Wavelog to run stably, I'll probably stick with Log4OM. At least the DB on my Synology NAS runs without any problems. One more addition - I have set up some Raspberry PIs, Ubuntu servers e.g. for my WebSDR and PhantomSDR Plus and now have a certain Linux routine - and have got everything up and running so far - but only because I document everything down to the last detail. Best 73 de HB9RYZ, Wolfgang

**Comment by HB9RYZ** [comment]
After a phone call with Fabian (HB9HIL) and his remote session support, it turned out that the Linux kernel of my Synology NAS does not meet the requirements for Portainer (see above post from hellofinch) and so I canceled the Wavelog project with my Synology NAS. Fabian created an account for me in his personal instance - many thanks Fabian :-)
